integration-cli/docker_cli_rm_test.go

```diff
@@ -19,6 +20,9 @@ func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
 	}
 	defer os.RemoveAll(tempDir)
 
+	out, _, _ := dockerCmdWithStdoutStderr(c, "ps", "-a")
+	fmt.Printf("out: %s", out)
```

Comment:
The idea is to print for all cases, so that we know what "ps -a" on a good case looks compared to a bad case.